_ml-commons-plugin/api.md

```diff
@@ -0,0 +1,741 @@
+---
+layout: default
+title: API
+has_children: false
+nav_order: 99
+---
+
+# ML Commons API 
+
+---
+
+#### Table of contents
+- TOC
+{:toc}
+
+
+---
+
+The Machine Learning (ML) commons API lets you train ML algorithms synchronously and asynchronously, make predictions with that trained model, train and predict with the same data set, and then store that model in an ML model index.
```

Comment:
Minor: we can remove "and then store that model in an ML model index" as this is more about implement details.